java/openpkw-services/src/test/java/org/openpkw/services/PeripherialCommitteeCsvParserTest.java

```diff
@@ -0,0 +1,67 @@
+package org.openpkw.services;
+
+import org.openpkw.mapping.PeripherialCommitteeLineMapper;
+import org.openpkw.model.entity.PeripheralCommittee;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+
+/**
+ * Created by wojciech.szostak on 15.08.2015.
+ */
+public class PeripherialCommitteeCsvParserTest {
+    private PeripherialCommitteeCsvParser csvParser;
+
+    @BeforeClass
+    public void init(){
+        csvParser = new PeripherialCommitteeCsvParser(
+                new PeripherialCommitteeLineMapper());
+    }
+
+    @Test
+    public void canParseFile() {
+        File tmpFile = new File("src/test/resources/pollstations.csv");
+        List<PeripheralCommittee> result = null;
+        try {
+            result = csvParser.parse(tmpFile);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
To jest test, więc plik powinien być w resourcach, ale wywaliłem `try/catch` bo też mi się nie podobało.